merkledag.go

```diff
@@ -281,30 +282,94 @@ func GetLinksWithDAG(ng ipld.NodeGetter) GetLinks {
 	}
 }
 
+// defaultConcurrentFetch is the default maximum number of concurrent fetches
+// that 'fetchNodes' will start at a time
+const defaultConcurrentFetch = 32
+
+// WalkOptions represent the parameters of a graph walking algorithm
+type WalkOptions struct {
```

Comment:
Let's make this private (or at least make the fields private). Yeah, I know _I_ don't usually do that but that was a mistake.

Reply:
Good point.